lang/src/org/partiql/lang/eval/physical/PhysicalPlanCompilerImpl.kt

```diff
@@ -380,13 +380,13 @@ internal class PhysicalPlanCompilerImpl(
     /**
      *  For operators which could return integer type, check integer overflow in case of [TypingMode.PERMISSIVE].
      */
-    private fun resolveIntConstraint(computeThunk: PhysicalPlanThunk, metas: MetaContainer): PhysicalPlanThunk =
+    private fun checkIntegerOverflow(computeThunk: PhysicalPlanThunk, metas: MetaContainer): PhysicalPlanThunk =
         when (val staticTypes = metas.staticType?.type?.getTypes()) {
             // No staticType, can't validate integer size.
             null -> computeThunk
             else -> {
-                when (evaluatorOptions.typingMode) {
-                    TypingMode.LEGACY -> {
+                when (evaluatorOptions.typedOpBehavior) {
```

Comment:
Similar comment to `EvaluatingCompiler.kt`. Thought this PR's change was only related to `typedOpBehavior` and not `typingMode`.